app/models/UserAnswers.scala

```diff
@@ -22,8 +22,8 @@ import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
 
 case class UserAnswers(
   eori: String,
-  contactDetails: Option[ContactDetails] = None,
   mrn: Option[MRN] = None,
+  contactDetails: Option[ContactDetails] = None,
```

Comment:
Nit: Don't think this was necessary, but nice touch ;-)

Reply:
No I didn't think so, but I wanted to keep things neat :P